apps/docs/components/Navigation/NavigationMenu/NavigationMenu.constants.ts

```diff
@@ -215,6 +215,7 @@ export const gettingstarted: NavMenuConstant = {
         { name: 'NuxtJS', url: '/guides/getting-started/quickstarts/nuxtjs' },
         { name: 'RedwoodJS', url: '/guides/getting-started/quickstarts/redwoodjs' },
         { name: 'Flutter', url: '/guides/getting-started/quickstarts/flutter' },
+        { name: 'Kotlin', url: '/guides/getting-started/quickstarts/kotlin' },
```

Comment:
```suggestion
        { name: 'Android Kotlin', url: '/guides/getting-started/quickstarts/kotlin' },
```

Reply:
Thanks @thorwebdev . According to your suggestion, it would be better if I name everything in my sample to be Android specifically. So I changed both menu title in quick starts and mobile tutorials section to Android, and also introduce Android icon to the repo. Hopefully it makes more sense.

Reply:
Not sure if it's a good idea to just name it "Android" as that could be literally anything. I think `Android Kotlin` would be more specific.

Reply:
Yeah that makes sense